spring-context/src/test/java/org/springframework/format/datetime/standard/InstantFormatterTests.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.format.datetime.standard;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+import java.text.ParseException;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.Random;
+import java.util.stream.Stream;
+
+import static java.time.Instant.MAX;
+import static java.time.Instant.MIN;
+import static java.time.ZoneId.systemDefault;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Andrei Nevedomskii
+ */
+@SuppressWarnings("ConstantConditions")
+class InstantFormatterTests {
+
+	private final InstantFormatter instantFormatter = new InstantFormatter();
+
+	@ParameterizedTest
+	@ArgumentsSource(ISOSerializedInstantProvider.class)
+	void should_parse_an_ISO_formatted_string_representation_of_an_instant(String input) throws ParseException {
+		Instant expected = DateTimeFormatter.ISO_INSTANT.parse(input, Instant::from);
+
+		Instant actual = instantFormatter.parse(input, null);
+
+		assertThat(actual).isEqualTo(expected);
+	}
+
+	@ParameterizedTest
+	@ArgumentsSource(RFC1123SerializedInstantProvider.class)
+	void should_parse_an_RFC1123_formatted_string_representation_of_an_instant(String input) throws ParseException {
+		Instant expected = DateTimeFormatter.RFC_1123_DATE_TIME.parse(input, Instant::from);
+
+		Instant actual = instantFormatter.parse(input, null);
+
+		assertThat(actual).isEqualTo(expected);
+	}
+
+	@ParameterizedTest
+	@ArgumentsSource(RandomInstantProvider.class)
+	void should_serialize_an_instant_using_ISO_format_and_ignoring_locale(Instant input) {
```

Comment:
How does this test ignore the locale any differently than the other test methods in this class?